gis/mapnik/Portfile

```diff
@@ -36,10 +36,13 @@ checksums           rmd160  14ec14c48ab790c8c3a2d30d51f8dd23b4dcd470 \
                     sha256  4b1352e01f7ce25ab099e586d7ae98e0b74145a3bf94dd365cb0a2bdab3b9dc2 \
                     size    10110103
 
+set boost_port      boost169
+set boost_path      ${prefix}/libexec/boost/${boost_port}
+
```

Comment:
Any reason to not use the boost PG here ? It should work just as well for boost 1.69 as the other versions ?

Reply:
Yep. First step was to simplify boost usage though, as the portfile was doing some wacky things it didn't need to...

Reply:
Ok thanks. Can you let me know when you are done pushing updates, and I will take a look and double check the builds are ok. I also have a few other ports I would like to move over to the boost PG, whilst we wait for this to be merged .